README.md

```diff
@@ -43,6 +43,11 @@ set :access_key_id, "CHANGETHIS"
 set :secret_access_key, "CHANGETHIS"
 ```
 
+### Capistrano 3
```

Comment:
To further avoid confusion I would split the configuration for Capistrano 2 too (or change the default suggested configuration to Capistrano 3)